src/ensembl/src/content/app/browser/Browser.tsx

```diff
@@ -253,9 +255,10 @@ export const Browser: FunctionComponent<BrowserProps> = (
   }, [props.chrLocation]);
 
   const closeTrack = () => {
-    if (props.drawerOpened === false) {
+    if (isDrawerOpened) {
```

Comment:
But now it’s the opposite of what is deleted above?

Reply:
Ah nice spot. Will fix it 😄